### DISCUSSION TITLE
bug

### DISCUSSION THREAD

**Discussion by Xangaranga3D** [community/community#120854] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![1713878635465](https://github.com/community/community/assets/125196564/bed2ed05-8fc2-4b87-b035-143ca0e1adff)

**Comment by ebndev** [comment]
Hi @Xangaranga3D,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific issue, you might want to try asking for help somewhere that focuses on Creality support. GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this.

Best of luck!
